db/migrate/20180620124108_use_utf8_mb4_for_comments.rb

```diff
@@ -0,0 +1,11 @@
+class UseUtf8Mb4ForComments < ActiveRecord::Migration[5.1]
```

Comment:
By default we are using sqlite.
But this migration does not work with sqlite:
`SQLite3::SQLException: near "CHARSET": syntax error: ALTER TABLE comments CHARSET utf8mb4 COLLATE utf8mb4_unicode_ci`
Also, in sqlite emojis are aready working.
Does it make sense to make this a mysql only migration?
Maybe with https://stackoverflow.com/questions/1628608/how-do-i-check-the-database-type-in-a-rails-migration